study-builder/studies/osteo/study.conf

```diff
@@ -1199,7 +1204,6 @@
         }
       ]
     },
-    # loved-one workflow
```

Comment:
Can we keep this comment? I kind of like it since it gives some context when scanning the file.